daemon/cmd/daemon_main.go

```diff
@@ -1454,6 +1464,7 @@ func initEnv(cmd *cobra.Command) {
 			log.Warn("netfilter-compatible-mode is enabled only on 5.4 and higher versions of kernel.")
 		}
 	}
+
```

Comment:
Extra space?

Reply:
Don't think so, this just separates the if block from the final bracket of the highest scope.